packages/opentelemetry/lib/index.js

```diff
@@ -0,0 +1,89 @@
+const { diag, DiagLogLevel } = require('@opentelemetry/api');
+const {
+	getNodeAutoInstrumentations
+} = require('@opentelemetry/auto-instrumentations-node');
+const { Resource } = require('@opentelemetry/resources');
+const opentelemetrySDK = require('@opentelemetry/sdk-node');
+const {
+	SemanticResourceAttributes
+} = require('@opentelemetry/semantic-conventions');
+const appInfo = require('@dotcom-reliability-kit/app-info');
+const {
+	OTLPTraceExporter
+} = require('@opentelemetry/exporter-trace-otlp-proto');
+const { NoopSpanProcessor } = require('@opentelemetry/sdk-trace-base');
+const logger = require('@dotcom-reliability-kit/logger');
+// OpenTelemetry Traces Endpoint to which to export traces (e.g. http:localhost:4318/v1/traces).
```

Comment:
**suggestion:** left in by accident?

```suggestion

```

Reply:
Yes sorry - Was trying to figure out the JSDoc nitpicky issue  - will remove thanks!